daemon/info_unix.go

```diff
@@ -58,28 +58,28 @@ func (daemon *Daemon) fillPlatformInfo(v *system.Info, sysInfo *sysinfo.SysInfo,
 	v.InitCommit.ID = "N/A"
 
 	if _, _, commit, err := parseDefaultRuntimeVersion(&cfg.Runtimes); err != nil {
-		log.G(context.TODO()).Warnf(err.Error())
+		log.G(ctx).Warnf(err.Error())
 	} else {
 		v.RuncCommit.ID = commit
 	}
 
-	if rv, err := daemon.containerd.Version(context.Background()); err == nil {
+	if rv, err := daemon.containerd.Version(ctx); err == nil {
 		v.ContainerdCommit.ID = rv.Revision
 	} else {
-		log.G(context.TODO()).Warnf("failed to retrieve containerd version: %v", err)
+		log.G(ctx).Warnf("failed to retrieve containerd version: %v", err)
 	}
 
 	v.InitBinary = cfg.GetInitPath()
 	if initBinary, err := cfg.LookupInitPath(); err != nil {
-		log.G(context.TODO()).Warnf("failed to find docker-init: %s", err)
+		log.G(ctx).Warnf("failed to find docker-init: %s", err)
```

Comment:
Same as above

Reply:
Same, this is not passing down a context (there is no need to)